incubator/webpagetest-server/README.md

```diff
@@ -0,0 +1,213 @@
+| `ec2Locations.userData.ec2_key`      | EC2 API key with create permissions        | `nil`                                                      |
+| `ec2Locations.userData.ec2_secret`   | Secret portion of the above key            | `nil`                                                      |
+| `ec2Locations.userData.headless`     | API Only Flag, 0=has user UI               | `0`                                                        |
+| `ec2Locations.userData.'EC2.default'`| default testing location when using AMI's  | `eu-west-1`                                                |
+| `serviceType`                        | Kubernetes Service type                    | `LoadBalancer`                                             |
+| `healthcheckHttps`                   | Use https for liveliness and readiness     | `false`                                                    |
+| `ingress.enabled`                    | Enable ingress controller resource         | `false`                                                    |
+| `ingress.hosts.[0]`                  | host header for access                     | `web-page-test.local`                                      |
+| `ingress.hosts.annotations`          | Annotations for the host's ingress records | `[]`                                                       |
+| `ingress.tls`                        | Utilize TLS backend in ingress             | `false`                                                    |
+| `ingress.tls.[0].secretName`         | TLS Secret (certificates) name             | `web-page-test.local-tls`                                  |
+| `ingress.tls.[0].hosts`              | host headers to get ssl certs for          | `[web-page-test.local]`                                    |
+| `agentIngress.*`                     | As ingress above, but used for WPT agents  | `false`                                                    |
+| `nodeSelector`                       | Node labels for pod assignment             | `{}`                                                       |
+
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```console
+$ helm install --name my-release \
+  --set ingress.enabled=true,ingress.hosts.[0]=web-page-test.local \
+    incubator/webpagetest-server
+```
+
+Alternatively, a YAML file that specifies the values for the above parameters can be provided while installing the chart. For example,
+
+```console
+$ helm install --name my-release -f values.yaml incubator/webpagetest-server
+```
+
+> **Tip**: You can use the default [values.yaml](values.yaml)
+
+
+## Ingress(es)
+
+This chart provides support for ingress resources. If you have an
+ingress controller installed on your cluster, such as [nginx-ingress](https://kubeapps.com/charts/stable/nginx-ingress)
+or [traefik](https://kubeapps.com/charts/stable/traefik) you can utilize
+the ingress controller to service your WordPress application.
```

Comment:
Wordpress? Seems like a copy and paste error. Wordpress shows up a few times in this PR.